example/android/build.gradle

```diff
@@ -6,8 +6,8 @@ buildscript {
     ext {
         buildToolsVersion = "31.0.0"
         minSdkVersion = 21
-        compileSdkVersion = 31
-        targetSdkVersion = 31
+        compileSdkVersion = 33
```

Comment:
Hmmm, why did we have to bump the versions here? Are we sure we want to do that?

Reply:
I updated an official android library (androidx), and that required an sdk bump. The min sdk version is still the same though, so it's not a breaking change.